docs/logs.md

```diff
@@ -2,4 +2,170 @@
 sidebar_position: 9
 ---
 
-# Logs
\ No newline at end of file
+# Logs Overview 📊
+
+## Query Search Interface Overview 🔍
+
+- Overview of the query search interface supporting SQL, SPL, and LogQL queries.
+
+
+
```

Comment:
Save video in assets folder in this repo, and show href link here with Video as text and relative file path (as you are providing for images) as link